Mathlib/Data/List/MinMax.lean

```diff
@@ -449,6 +449,16 @@ theorem minimum_of_length_pos_le_iff (h : 0 < l.length) :
     minimum_of_length_pos h ≤ b ↔ l.minimum ≤ b :=
   le_maximum_of_length_pos_iff (α := αᵒᵈ) h
 
+theorem maximum_of_length_pos_mem (h : 0 < l.length) :
+    maximum_of_length_pos h ∈ l := by
+  apply maximum_mem
+  simp only [coe_maximum_of_length_pos]
+
+theorem minimum_of_length_pos_mem (h : 0 < l.length) :
+    minimum_of_length_pos h ∈ l := by
```

Comment:
As a final trick; you should be able to prove this using the `(α := αᵒᵈ)` trick the minimum lemmas in the rest of the file use

Reply:
Thanks! it is done.